x-pack/legacy/plugins/canvas/i18n/transitions.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const TransitionStrings = {
+  fade: {
+    getDisplayName: () => () =>
```

Comment:
These are all functions returning functions that return the text, instead of functions that return the text, so none of the text is showing up.  

![image](https://user-images.githubusercontent.com/515272/66498520-365e9500-ea8c-11e9-8b18-09e0ab6f9eda.png)

Reply:
Yep. Good catch. Doh.